detection_rules/schema.py

```diff
@@ -100,6 +104,13 @@ class SeverityMapping(jsl.Document):
     severity = jsl.StringField(required=False)
 
 
+class ThresholdMapping(jsl.Document):
+    """Threshold mapping."""
+
+    field = jsl.StringField(required=True)
```

Comment:
For threshold rules, the `field` to aggregate events by is optional, the value/count is required.

```suggestion
    field = jsl.StringField(required=False)
```